### ISSUE TITLE
'Sum' measurement only available for Nucleus in Cell detection

### ISSUE THREAD

**Issue by lacan** [qupath/qupath#1454]
## Bug report

**Describe the bug**
After running QuPath's cell detection on a multichannel fluorescent image with Channel names "Nuclei" and "Protein X", the measurements
'Nucleus: Nuclei sum` and `Nucleus: Protein X sum` are computed.

However the following measurements are missing
'Cell: Nuclei sum` and `Cell: Protein X sum`
'Cytoplasm: Nuclei sum` and `Cytoplasm: Protein X sum`.

**To Reproduce**
1. Open a fluorescence image
2. Use Cell detection with a non zero expansion
3. Look a the resulting measurements for each cell

**Expected behavior**
All compartments should have `sum` measurement for all channels or not have it at all

**Screenshots**
![image](https://github.com/qupath/qupath/assets/319932/9bbf5da2-b5bb-4cae-a0b1-4efa96e2ec5a)
Image with cells detected and the detectiom measurements showing the columns containing `sum`
Thenon `Nucleus` compartments do not have a `sum` measurement

**Desktop (please complete the following information):**
 - OS: Windows 11
 - QuPath Version: 0.5.0

All the best

**Comment by petebankhead** [comment]
I'm not sure about this. It has been a long time, but I think the reason I excluded sum from non-nucleus measurements is because it is common for people to train classifiers using all features.

Because cell expansion is only distance-based, I felt that sum measurements outside the nucleus could be extremely unreliable. Also, the sum depends upon the resolution at which the measurement is calculated; for the default cell detection, this is the detection resolution, but that might not be what the user expects.

The need for a nucleus sum, however, seemed sufficiently common that it should be included - and the nucleus ROI is likely to be more reliable, which reduces the problems slightly.

I realise it's a judgement call and people may disagree. I'm reluctant to change the built-in cell detection, since the current behavior hasn't changed across releases and any modifications can impact classifiers. I'd rather we focus on moving away to alternative cell detection methods, and decouple measurement from detection to improve flexibility.

**Comment by MichaelSNelson** [comment]
You can also calculate it fairly quickly across all cells by using
`cytoplasmic mean * (cell Area - nuclear area)`

**Comment by lacan** [comment]
Yes I understand the logic behing it and I think @petebankhead's logic is sound. intensty sum means little when the cytoplam and cells are not "real" structures but geometrical constructs. And thanks @MichaelSNelson. I ended up using the mean intensity measurement to get what I needed and your explanation was very welcome!
